### DISCUSSION TITLE
Summit 2023 instructions

### DISCUSSION THREAD

**Discussion by olzama** [delph-in/docs#130] [Announcements]
Dear @delph-in/participants :

The summit starts tomorrow! 

Here's the information for the participants not coming in person: 

https://zoom.us/j/4768685765?pwd=NnZqQmx5QjcyNm94b1BuRUR3NUVkZz09

Meeting ID: 476 868 5765
Passcode: 459442

Those coming in person: please arrive by 10:15 or so so that we can start on time.

Once in the courtyard of the [rectorate](https://goo.gl/maps/65A15Me3tu2vDKQa7), head to the main entrance, pictured below:

![rectorate](https://github.com/delph-in/docs/assets/10963114/60719dec-35a0-4e1f-bd64-638f20bb6699)

Head to the "2nd floor" (which is the third floor if you start counting from the ground). You need the room called "Consello de Goberno"; once you are up on the 2nd floor, the entrance to the room is just slightly to the right from the staircase. Someone will be there greeting people until 10:25 or so.

**Comment by john-a-carroll** [comment]
Dear @delph-in/participants 

Many thanks, Olga, for your clear instructions.

And now a plea from the PC co-chairs: if you are presenting, please upload a PDF of your slides to https://github.com/delph-in/docs/tree/main/summits/2023 and then link them to your session on the [schedule](https://github.com/delph-in/docs/wiki/GaliciaSchedule) webpage. (Note that you need to be logged in to GitHub to be able to do both of these steps).

To avoid potential problems with switching data projector cables and Zoom connections, we strongly recommend that all presentations be given from the laptop provided in the room. In this case, it will help a lot if your slides are linked from the schedule webpage.

Thanks in advance for your cooperation!

John & Petter

**Comment by olzama** [comment]
@delph-in/participants :

I believe the first food will be served during the first coffee break, which is at 12:00. Please plan accordingly! You can get something quick to eat, or at least some croissant, in the Old Town.

I use this opportunity to second John's plea for slide uploads. This includes **Site updates**!

**Comment by john-a-carroll** [comment]
Good evening @delph-in/participants 

Thank you for filling in your interests in the SIGs/tutorials spreadsheet. Petter and I have now scheduled them - please see GaliciaSchedule.

We've ended up with 9 SIGs/tutorials, so all are plenary apart from those in the first session on Thursday morning. We will use the two rooms just for that session, and then reduce to one. We'll confirm Zoom links before the morning.

All the best,

John & Petter

**Comment by olzama** [comment]
Which Thursday morning session will be in which room?

On Wed, Jun 28, 2023 at 7:41 PM John Carroll ***@***.***>
wrote:

> Good evening @delph-in/participants
> <https://github.com/orgs/delph-in/teams/participants>
>
> Thank you for filling in your interests in the SIGs/tutorials spreadsheet.
> Petter and I have now scheduled them - please see GaliciaSchedule.
>
> We've ended up with 9 SIGs/tutorials, so all are plenary apart from those
> in the first session on Thursday morning. We will use the two rooms just
> for that session, and then reduce to one. We'll confirm Zoom links before
> the morning.
>
> All the best,
>
> John & Petter
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/delph-in/discussions/130#discussioncomment-6307460>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACTURKQA72FVMDLMO6YBLZ3XNRUERANCNFSM6AAAAAAZTHIE74>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
Olga

**Reply by john-a-carroll** [reply]
It seems that several people prefer the room that we were in today, [Sala de Xuntas](https://www.udc.es/es/goberno/equipo_reitoral/xerencia/servizos/control-interno/inventario-e-patrimonio/patrimonio/sala_xuntas/), so that will become our main room, starting with the 'Full Forest Treebanker (FFTB)' tutorial on Thursday morning. We will use [Sala del Consello de Goberno](https://www.udc.es/es/goberno/equipo_reitoral/xerencia/servizos/control-interno/inventario-e-patrimonio/patrimonio/sala_consello_goberno/) only for the 'Tuning a grammar to improve parsing efficiency' SIG.

Zoom links will be as follows:

https://zoom.us/j/4768685765?pwd=NnZqQmx5QjcyNm94b1BuRUR3NUVkZz09
The 'Full Forest Treebanker (FFTB)' tutorial and all SIGs and tutorials thereafter

https://washington.zoom.us/j/91223884636
The 'Tuning a grammar to improve parsing efficiency' SIG only

See you tomorrow at 10:30,

John

**Comment by emilymbender** [comment]
It should not require one -- but I would appreciate it if someone could try logging in to make sure it works without my presence.

**Reply by john-a-carroll** [reply]
That link works for me as-is.

**Comment by john-a-carroll** [comment]
I'm afraid that a dead laptop has forced us to swap sessions 1 and 3 on Friday. The changes to [GaliciaSchedule](https://github.com/delph-in/docs/wiki/GaliciaSchedule) are as follows:

10:30-12:00  DELPH-IN documentation & the new DELPH-IN documentation system
15:30-17:00  Searching DELPH-IN corpora & the Linguistic Type Database (LTDB)

John

**Comment by olzama** [comment]
Those joining for the Friday dinner: meet at 20:30 in the restaurant. It is called O Tanagra and is located sort of between streets:

https://goo.gl/maps/UMdr4hRi2hWvyRxE6

**Comment by olzama** [comment]
@delph-in/participants I published links to discussions [here](https://github.com/delph-in/private-media-etc/discussions/2). **You need to be logged in** in order to access, otherwise you will see a 404 error. Please let me know if there are any problems with any of the specific links! Once again, if you see a 404 error when you click on the main link, means you are not logged in, probably.
